### PR TITLE
Added the slack channel to the contributing page of the website (Fixes #893)

### LINKED ISSUE #893: Add a link and info about the Lucene.NET Slack channel
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the documentation issue

We set up a Slack channel (provided by Apache) some time ago, but it hasn't found its way yet onto our [Contributing](https://lucenenet.apache.org/contributing/index.html#getting-involved) section.

OpenNLP (another Apache project) has their Slack channel as an official way to contact them. Here is what their page looks like: https://opennlp.apache.org/get-involved.html

### Additional context

The Apache docs for Slack are here: https://infra.apache.org/slack.html.

The channel is located at: [#lucenenet-dev](https://the-asf.slack.com/archives/C03FFRQQ2RM)

### PR DESCRIPTION
**Pull request by jaivardhan-bhola**
- [x] You've read the [Contributor Guide](https://github.com/apache/lucenenet/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://www.apache.org/foundation/policies/conduct.html).
- [x] You've included unit or integration tests for your change, where applicable.
- [x] You've included inline docs for your change, where applicable.
- [x] There's an open issue for the PR that you are making. If you'd like to propose a change, please [open an issue](https://github.com/apache/lucenenet/issues/new/choose) to discuss the change or find an existing issue.

Summary of the changes (Less than 80 chars)

Added the slack channel to the contributing page of the website.

Fixes #893

### CODE REVIEW

File: websites/site/contributing/index.md [hunk 1]
```diff
@@ -40,6 +40,14 @@ We recommend you join both the [user and dev mailing lists](https://cwiki.apache
 
 see **[mailing lists](xref:contributing/mailing-lists)**
 
+### Slack Channel
+Another way of contacting Lucene.net committers and PMC members is the [#lucenenet-dev](https://the-asf.slack.com/archives/C03FFRQQ2RM) Slack channel. This is a great place to meet and get guidance on contributions.
+
+Further details on using the ASF Slack channels and associated expectations can be found on the related [ASF infra page](https://infra.apache.org/slack.html).
+
+Note
```
Comment (NightOwl888) [inline websites/site/contributing/index.md:48]: Please change this to use the docfx format so it looks nice in the docs:

```
> [!NOTE]
> The ASF Slack requires an @apache.org email address to join as a full member. Users without an @apache.org email address have to be invited.
```

**Review by NightOwl888 — CHANGES_REQUESTED** [review]
Thanks for the PR. There is just one minor change to make and this will be good to merge.

File: websites/site/contributing/index.md (hunk 1 above)
Comment (NightOwl888) [reply websites/site/contributing/index.md:48]: It will format the note like this:

image

**Comment by jaivardhan-bhola** [comment]
I have done the changes you asked for

**Review by NightOwl888 — APPROVED** [review]

**Pull request merged**